src/test/java/no/digipost/security/cert/CertificateValidatorTest.java

```diff
@@ -337,8 +349,8 @@ public void doRealOcspLookup() throws Exception {
 
 
         try (CloseableHttpClient realClient = HttpClient.create()) {
-            X509Certificate certificate = Certificates.revoked();
-            X509Certificate issuer = Certificates.revokedIssuer();
+            X509Certificate certificate = BUYPASS_SEID_2_CERT;
+            X509Certificate issuer = BUYPASS_SEID_2_ISSUER;
```

Comment:
Bare sette disse tilbake til _hva det var_, så slipper vi litt diff-noise?
```suggestion
            X509Certificate certificate = Certificates.revoked();
            X509Certificate issuer = Certificates.revokedIssuer();
```
<img width="443" alt="hva-det-var" src="https://user-images.githubusercontent.com/174823/128882335-bd72e30e-0703-4cf8-976b-cc2271e612ab.png">


_Props for the ice rocker_ hvis noen tar disse referansene her altså. Sterkt meme-game på denne PR-en nå, i all beskjedenhet!
<img width="443" alt="props" src="https://user-images.githubusercontent.com/174823/128882423-e944b54d-2c94-4264-bf87-710c3bcbc1c0.png">

Reply:
Haha, jeg henger ikke med hvertfall 😛